ppml/trusted-realtime-ml/scala/docker-graphene/kubernetes/README.md

```diff
@@ -1,6 +1,6 @@
 # Trusted Realtime ML for Kubernetes
 
-## Deploy the Intel SGX Device Plugin for Kubenetes
+## Deploy the Intel SGX Device Plugin for Kubernetes
 
 Please refer to the document [here][devicePluginK8sQuickStart].
 
```

Comment:
Line 74 https://bigdl.readthedocs.io/en/latest/doc/PPML/QuickStart/deploy_intel_sgx_device_plugin_for_kubernets.html also need be modified